### PR TITLE
Update Routing Tests to work with the new URL formats

### PR DESCRIPTION
**Pull request by kiran-mastek**
Update the Dashboard and OrderForm Journey to Include the ODS Code in the URL
Test fixes

Story: [AB#14286](https://buyingcatalog.visualstudio.com/c5f97979-5b03-4d10-ba8d-871d0526b408/_workitems/edit/14286)
Task: [AB#14360](https://buyingcatalog.visualstudio.com/c5f97979-5b03-4d10-ba8d-871d0526b408/_workitems/edit/14360), [AB#14365](https://buyingcatalog.visualstudio.com/c5f97979-5b03-4d10-ba8d-871d0526b408/_workitems/edit/14365)

### CODE REVIEW

File: app/pages/sections/order-items/additional-services/dashboard/contextCreator.test.js [hunk 1]
```diff
@@ -7,21 +7,22 @@ import { baseUrl } from '../../../../../config';
...
     ${''}                                   | ${additionalServicesUrl}                                    | ${mockSelectedPricePatientType} | ${''}
     ${`${somefakeUrl}/date`}                | ${`${somefakeUrl}/date`}                                    | ${mockSelectedPricePatientType} | ${''}
     ${`${somefakeUrl}/additional-services`} | ${`${somefakeUrl}/additional-services`}                     | ${mockSelectedPricePatientType} | ${''}
-    ${`${somefakeUrl}/recipients`}          | ${`${baseUrl}/organisation/${orderId}/additional-services`} | ${mockSelectedPricePatientType} | ${''}
+    ${`${somefakeUrl}/recipients`}          | ${`${baseUrl}/organisation/${odsCode}/order/${orderId}/additional-services`} | ${mockSelectedPricePatientType} | ${''}
     ${`${somefakeUrl}/neworderitem`}        | ${onDemandPriceUrl}                                         | ${mockSelectedPriceOnDemandType}| ${''}
     ${`${somefakeUrl}/neworderitem`}        | ${dateUrl}                                                  | ${mockSelectedPricePatientType} | ${''}
```
Comment (bjss-chris) [inline app/pages/sections/order-items/additional-services/dashboard/contextCreator.test.js:27]: Can we fix the alignment of the vertical bars in the table?

File: app/pages/sections/order-items/associated-services/order-item/contextCreator.test.js [hunk 2]
```diff
@@ -9,20 +9,23 @@ describe('associated-services order-item contextCreator', () => {
...
     const mockAssociatedServicePrices = { prices: [{ priceId: 29 }, { priceId: 30 }] };
     const somefakeUrl = 'https://some.url.co.uk/order-id';
+    const odsCode = '03F';
     it.each`
     senderUrl                               |  expectedUrl                           | associatedServicePrices
     ${`${somefakeUrl}/associated-service`}  | ${`${somefakeUrl}/associated-service`} | ${''}   
     ${`${somefakeUrl}/price`}               | ${`${somefakeUrl}/price`}              | ${''}  
     ${`${somefakeUrl}/associated-services`} | ${`${somefakeUrl}/associated-services`}| ${''}   
-    ${`${somefakeUrl}/neworderitem`}        | ${`${baseUrl}/organisation/${orderId}/associated-services/select/associated-service/price`} |${mockAssociatedServicePrices}   
+    ${`${somefakeUrl}/neworderitem`}        | ${`${baseUrl}/organisation/${odsCode}/order/${orderId}/associated-services/select/associated-service/price`} |${mockAssociatedServicePrices}
```
Comment (bjss-chris) [inline app/pages/sections/order-items/associated-services/order-item/contextCreator.test.js:18]: Can we fix the alignment of the vertical bars in the table?

**Review by bjss-chris — CHANGES_REQUESTED** [review]
Looks good. Just some column alignment issues in a couple of the unit tests that would be good to fix.

File: app/pages/sections/order-items/additional-services/dashboard/contextCreator.test.js (hunk 1 above)
Comment (kiran-mastek) [reply app/pages/sections/order-items/additional-services/dashboard/contextCreator.test.js]: done

File: app/pages/sections/order-items/associated-services/order-item/contextCreator.test.js (hunk 2 above)
Comment (kiran-mastek) [reply app/pages/sections/order-items/associated-services/order-item/contextCreator.test.js]: done

**Review by bjss-chris — APPROVED** [review]

**Pull request merged**